drivers/gke/gke_driver.go

```diff
@@ -474,20 +666,38 @@ func (d *Driver) Update(ctx context.Context, info *types.ClusterInfo, opts *type
 		}
 	}
 
+	if newState.NodePool.Autoscaling.Enabled {
+		if newState.NodePool.Autoscaling.MinNodeCount < 1 || newState.NodePool.Autoscaling.MaxNodeCount < newState.NodePool.Autoscaling.MinNodeCount {
+			return nil, fmt.Errorf("minNodeCount in the NodePool must be >= 1 and <= maxNodeCount")
```

Comment:
This logic belongs inside the `validate` func.

Reply:
Yeah, it makes sense, if it is necessary to make this logic belong inside the `validate` func, I will update it immediately.

Reply:
Updated! Please review it again, thanks.